tools/ansible/roles/dockerfile/files/wait-for-migrations

```diff
@@ -25,7 +24,7 @@ wait_for() {
     local check=1
 
     while true; do
-        log_message "Attempt ${attempt} of ${ATTEMPTS}"
+        log_message "Attempt ${attempt}"
```

Comment:
I don't get it. These messages about attempts (not just this but another one a little later) don't make any sense to me after this change.

Reply:
you are right im gonna still increment attempt at the end